secant/Features/Onboarding/OnboardingStore.swift

```diff
@@ -11,38 +11,54 @@ import ComposableArchitecture
 
 struct OnboardingStep: Equatable, Identifiable {
     let id: UUID
+    let title: String
     let description: String
-    let imageName: String
 }
 
 struct OnboardingState: Equatable {
     var steps: IdentifiedArrayOf<OnboardingStep> = Self.onboardingSteps
     var index = 0
     var offset: CGFloat = .zero
```

Comment:
[minor]

[from offline chat]
Would it make sense to derive this offset from the `index` and `steps.count`? (and possibly screen dimensions?)